wordpress-mobile/WordPress-iOS
# Media Library: Remove async feature flag 🎉

This PR removes the feature flag for async in the media library.

**To test:**

* In the media library, test each upload option and check that they work asynchronously (no HUD).
* Ensure that progress is shown correctly for each upload.
* Check that uploads can be cancelled.
* Check that failed uploads show the correct state and can be retried.
